Dockerfile

```diff
@@ -29,7 +29,7 @@ ENV HELM_DEBUG 1
 # WARNING: If you are changing any versions here, update it in the reference.conf
 ENV TERRA_APP_SETUP_VERSION 0.0.2
 ENV TERRA_APP_VERSION 0.3.0
-ENV GALAXY_VERSION 1.0.0
+ENV GALAXY_VERSION 1.1.0
```

Comment:
huh... whats the correlation between this number and the one in terra-UI (21.05)

Reply:
This is technically the `galaxykubeman` helm chart version, which references the Galaxy _app_ version 21.05..

Reply:
would be nice to add the galaxy app version as a comment